fdbserver/DataDistributionTracker.actor.cpp

```diff
@@ -650,12 +650,63 @@ ACTOR Future<Void> fetchShardMetrics( DataDistributionTracker* self, GetMetricsR
 	return Void();
 }
 
+
+ACTOR Future<Void> fetchShardMetricsList_impl( DataDistributionTracker* self, GetMetricsListRequest req ) {
+	try {
+		loop {
+			// no shard identifier as of yet, use simple numbering system
+			// also used to control shard limit
+			int shardNum = 1;
+			// list of metrics, regenerate on loop when full range unsuccessful
+			Standalone<RangeResultRef> result;
+			Future<Void> onChange;
+			for( auto t : self->shards.intersectingRanges( req.keys ) ) {
+				auto &stats = t.value().stats;
+				if( !stats->get().present() ) {
+					onChange = stats->onChange();
+					break;
+				}
+				std::string keyRanges = "Begin: " + t.begin().toString() + ", End: " + t.end().toString();
+				result.push_back_deep(
+				    result.arena(), KeyValueRef(KeyRef(std::to_string(shardNum)),
+				                                ValueRef(keyRanges + ", " + t.value().stats->get().get().toString())));
```

Comment:
We should reach a consensus amongst multiple people on what exactly the format or the keys and values should be (should we use JSON? A unique shard identifier as key? etc.) Once this issue is resolved and some other review comments are addressed, this PR can be upgraded to a non-draft PR.